src/Npgsql/Netstandard20/ReadOnlySpanOfCharExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace Npgsql.Netstandard20
+{
+    static class ReadOnlySpanOfCharExtensions
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static int ParseInt(this ReadOnlySpan<char> span)
```

Comment:
I'd personally not do this extension method over such a common type (which will appear in Intellisense) just for one call site...

Reply:
Never mind, was already merged :) Maybe I'll clean up later.

Reply:
I'm pretty sure this will occur more often once we get used to using `ReadonlySpan<char>` for parsing things (as long as we have to stick with .NET Standard 2.0).

I guess the only reason why there isn't a shim already is the fact that you can't create a static extension method over Int32 (and perhaps because it sneaks in an allocation where you're actually trying to avoid it).

Reply:
Yeah, it's all internal anyway, so doesn't matter so much either way.